app/sections/view-supplier-asserted-integrations/macro.test.js

```diff
@@ -0,0 +1,112 @@
+import request from 'supertest';
+import cheerio from 'cheerio';
+import { createTestHarness } from '../../testUtils/testHarness';
+
+const macroWrapper = `{% from './sections/view-supplier-asserted-integrations/macro.njk' import viewSupplierAssertedIntegrations %}
+                        {{ viewSupplierAssertedIntegrations(params) }}`;
+
+describe('view-supplier-asserted-integrations', () => {
+  it('should render the supplier asserted integrations answer if provided', (done) => {
```

Comment:
can you add tests for the opposite too e.g. should not render the supplier asserted integrations answer if not provided

Reply:
there's a test right under that one called 'should not render the supplier asserted integrations answer if not provided'

Reply:
sorry I must be going blind ! please add another for link and one for additional information

Reply:
is this not overkill? Additional information is not user input so if there's no link there's no section

Reply:
is that because it is getting passed in as a component?

Reply:
ok I see now sorry I was taking it as face value rather than comparing to the macro - it is being passed in as a component so it will be fine 👍

Reply:
it's hardcoded: 
`innerComponent: viewDataText({
          dataTestId: 'view-data-text-additional-information',
          data: 'Supplier assured integrations are interoperability interfaces prepared by a supplier and are not specified or assured by the NHS.'
        }) + viewDataLink({
          dataTestId: 'view-question-data-text-link',
          data: params.section.answers['link']
        })`